### PR TITLE
CIGI-234: Create template for recommended section

### PR DESCRIPTION
**Pull request by johnbxu**
#### Description of changes
resolves CIGIHub/cigi-tickets#234

image

### CODE REVIEW

File: templates/articles/article_page.html [hunk 1]
```diff
@@ -72,4 +72,8 @@ <h2>Works Cited</h2>
     {% include "includes/about_the_author.html" with authors=self.authors %}
   {% endif %}
 
+  {% if self.recommended|length > 0 %}
```
Comment (gabrieltanchen) [inline templates/articles/article_page.html:75]: Nothing is showing up, I believe this needs to be `if self.recommended.all|length > 0`.

File: templates/includes/recommended.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<section class="recommended container-lg">
+  <div class="recommended-dot"></div>
+  <div class="recommended-vertical-line"></div>
+  <h2>Recommended</h2>
+  {% include "includes/footer_bracket.svg" %}
+
+  <div class="recommended-content-medium row">
+    {% for content in recommended %}
```
Comment (gabrieltanchen) [inline templates/includes/recommended.html:8]: Limit this to the first 3 in recommended.

File: templates/publications/publication_page.html [hunk 3]
```diff
@@ -91,6 +91,9 @@ <h3>
     {% include "includes/about_the_author.html" with authors=self.editors title_override="About the Editor" %}
   {% endif %}
 
+  {% if self.recommended|length > 0 %}
```
Comment (gabrieltanchen) [inline templates/publications/publication_page.html:94]: Update condition here as well.

**Review by gabrieltanchen — CHANGES_REQUESTED** [review]

**Comment by johnbxu** [comment]
Updated if and for conditions

**Review by gabrieltanchen — APPROVED** [review]

**Pull request merged**